tests/replication/slave/storage/_base.py

```diff
@@ -41,15 +43,17 @@ def setUp(self):
         self.event_id = 0
 
         server_factory = ReplicationStreamProtocolFactory(self.hs)
-        listener = reactor.listenUNIX("\0xxx", server_factory)
+        # XXX: mktemp is unsafe and should never be used. but we're just a test.
+        path = tempfile.mktemp(prefix="base_slaved_store_test_case_socket")
```

Comment:
This looks unrelated. Can you make it a separate PR please

Reply:
yes, it is unrelated, other than being pulled in to fix the tests whilst doing this PR.  given it's trivial, i'd prefer to not burn the time splitting it out at this point.